### ISSUE TITLE
Bug: When loading a network from database. If no relations are observed GRNsight crashes

### ISSUE THREAD

**Issue by Onariaginosa** [dondi/GRNsight#990]
When you load a network that has no regulatory connections between genes (no edges/links) from the database, it crashes GRNsight. I fixed it while working on #978 and pulled it out to a new issue for clarification. I cannot actually submit a PR for this fix quite yet because the export to excel functionality is not  fully functional at the moment. Once #978 is done, a PR will be submitted.

**Comment by dondi** [comment]
We don’t recall if this was verified—we can verify this alongside #995. After initializing a new database cluster but _before_ loading the schema, run GRNsight against the empty database to check if this issue is indeed addressed

**Comment by kdahlquist** [comment]
@ahmad00m says he checked this and confirmed that it was fixed.
